src/libpgmoneta/link.c

```diff
@@ -60,10 +63,9 @@ pgmoneta_link_manifest(char* base_from, char* base_to, char* from, struct art* c
    {
       goto done;
    }
-
    while ((entry = readdir(from_dir)))
    {
-      if (!strcmp(entry->d_name, ".") || !strcmp(entry->d_name, "..") || !strcmp(entry->d_name, "pg_tblspc"))
+      if (!strcmp(entry->d_name, ".") || !strcmp(entry->d_name, ".."))
```

Comment:
Is this ok ?

Reply:
Not sure what you mean, but it's tested, link function is able to penetrate through symlinks.